bin/git-cml

```diff
@@ -93,18 +93,22 @@ def add(args):
     Splits model parameters file into parameter groups on filesystem and stages files
     """
     repo = git_utils.get_git_repo()
-    model_path = git_utils.create_git_cml_model_dir(repo, args.file)
+    model_path = git_utils.get_relative_path_from_root(repo, args.file)
+    cml_model_dir = git_utils.get_git_cml_model_dir(repo, model_path, create=True)
 
     param_dict = checkpoints.PyTorchCheckpoint.from_file(args.file)
-    for (param, param_dir) in iterate_dict_and_dir_leaves(param_dict, model_path):
+    for (param, param_dir) in iterate_dict_and_dir_leaves(param_dict, cml_model_dir):
         param_file = os.path.join(param_dir, "params")
         file_io.write_tracked_file(param_file, param)
         git_utils.add_file(param_file, repo)
     
     for param_file in iterate_removed_params(param_dict, model_path):
```

Comment:
I think this for loop needs to be removed right?

Reply:
It should be present. It iterates over params in the model checkpoint.